src/components/CohortBuilder/Results/Toolbar/Toolbar.scss

```diff
@@ -22,27 +22,30 @@
   border-left: 1px solid #d7d8db;
   margin-right: 16px;
   height: 24px;
-}
 
-.cb-toolbar-info .ant-btn.ant-btn-link {
-  padding: 0;
-  height: auto;
-}
+  .cb-toolbar-item {
+    align-items: center;
+    display: flex;
+    margin: 0 10px 0 10px;
+    font-weight: 600;
+    font-family: Montserrat, sans-serif;
```

Comment:
We really have to fix the font. I think we have 2 different fonts atm in the kf-portal.